setup.py

```diff
@@ -34,6 +34,7 @@ def run_tests(self):
     "scipy >=0.19.1",
     "pims >=0.4.1",
     "tifffile >=2018.10.18",
+    "pandas",
```

Comment:
Should we give a specific minimum version? If so, how to choose one?

Reply:
I'm choosing to set the minimum pandas version as 2.0.0. 
It's not a hard technical requirement, but it will make future maintenance easier for us later if we restrict it to pandas version 2 only.